packages/types/src/networks.ts

```diff
@@ -156,4 +164,12 @@ export enum CoingeckoPlatform {
   Scroll = "scroll",
   Bitrock = "bitrock",
   Fraxtal = "fraxtal",
+  Gravity = "gravity-alpha",
+  Berachain = "berachain",
+  Abstract = "abstract",
+  Unichain = "unichain",
+  Story = "story",
+  Ink = "ink",
+  Taraxa = "taraxa",
+  Coti = "coti",
```

Comment:
this platform doesnt exists
make sure it is in https://api.coingecko.com/api/v3/coins/list?include_platform=true under platforms key